src/structures/Message.js

```diff
@@ -809,6 +809,14 @@ class Message extends Base {
     return this.edit({ attachments: [] });
   }
 
+  /**
+   * Indicates whether this message is a partial.
```

Comment:
```suggestion
   * Indicates whether this message is partial.
```

Reply:
There are a lot of occurrences that say "a partial" instead of just "partial." Should those be changed? 🤔